tdei-api-gateway.json

```diff
@@ -181,30 +239,60 @@
                         "AuthorizationToken": []
                     }
                 ]
-            },
+            }
+        },
+        "/api/v1/osw/upload/{tdei_project_group_id}": {
```

Comment:
how is the mobility service ID taken? Use case says: For WSDOT project_group, KCM Mobility service, OSW data is being uploaded. It seems like you assumed only project_group is necessary for OSW. Change.

Reply:
@sureshgaussian , Currently we do not have 'service_id' for OSW resource. If anything missing from requirement let me know I will include the change.

Reply:
@MashB , Use case 1 shows proj_group as WSDoT, mobility_service as KCM. So, we should add mobility_service for OSW.